sdk/translation/azure-ai-translation-document/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Release History
+## 1.0.0-beta.2 (2024-06-18)
+
+### Other Changes
+- Re-release of 1.0.0-beta.1
```

Comment:
[README](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/translation/azure-ai-translation-document/README.md#adding-the-package-to-your-product) should be updated as well.